content/en/blog/_posts/0000-00-00-kube-proxy-non-privileged.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: blog
+title: "Kubernetes 1.29: Support for Running kube-proxy in a Non-privileged Container"
+date: 0000-00-00
+slug: kube-proxy-non-privileged
+---
+
+**Author**: Lars Ekman
+
+This post describes how the `--init-only` flag to `kube-proxy`, added in K8s v1.29, can
+be used to perform configuration that requires privileged mode in an
+initContainer, while the main `kube-proxy` container may run with a
+stricter securityContext.
```

Comment:
maybe flip that around to put the cool part first: "can be used to run the main kube-proxy container in a stricter securityContext by performing the configuration that requires privileged mode in a separate initContainer"

Reply:
I also removed the "added in K8s v1.29", since it's now in the title.